src/gui/components/props/input/input.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { Component, PropTypes } from 'react'
+import classnames from 'classnames'
+import styles from './input.styles.scss'
+
+class Input extends Component {
+
+  handleChange = ({ target }) => {
```

Comment:
I don't like that notation, I don't understand why so much people do this :(
This is a bit of overhead for something simple :
 - A class
 - Inheritence
 - autobinded method

Just to avoid the event.target.value in the container in other hand